### PR TITLE
Dfuse get account API

### PR DESCRIPTION
**Pull request by gangadharjannu**

### CODE REVIEW

File: back-end/routes/chain/get_account.js [hunk 1]
```diff
@@ -1,27 +1,58 @@
...
+    .map(transaction => {
+      const transactionItem = transaction.lifecycle;
+      const accountName = transactionItem.execution_trace.action_traces[0].act.data.name;
+      return accountName;
+    });
+
+  if (!accountNamesInfo || accountNamesInfo.length === 0) {
+    res.status(404);
+    res.send({ message: `Not found account from dfuse with account name ${accountName}` });
+    return; // not sure if this is needed...
```
Comment (theblockstalk) [inline back-end/routes/chain/get_account.js:34]: should not be needed

File: back-end/routes/chain/get_account.js [hunk 2]
```diff
@@ -1,27 +1,58 @@
...
+  const accountName = req.body.account_name;
+  if (!accountName) {
     res.status(400);
     res.send({ message: 'req body should contain all the data!' });
     return;
   }
-
-  const accountDoc = await accountController.findOne({ accountName: req.body.account_name });
-  
+  // TODO: replace this with Dfuse API
```
Comment (theblockstalk) [inline back-end/routes/chain/get_account.js]: remove comment on line 14 and 16

File: back-end/routes/chain/get_account.js (hunk 1 above)
Comment (gangadharjannu) [reply back-end/routes/chain/get_account.js:34]: Done

File: back-end/routes/chain/get_account.js (hunk 2 above)
Comment (gangadharjannu) [reply back-end/routes/chain/get_account.js]: Done

**Pull request merged**